import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/AMPStaticProcessor.java

```diff
@@ -298,6 +298,13 @@ private void updateProject(JsonBean project, InternalDocument source, List<Field
 			case LIST:
 				processListDestinationProjects(source, valueMappings, project, mapping, sourceField, destinationField);
 				break;
+			 case ORGANIZATION:
```

Comment:
lets add a constant for funding organization and provider organization
Also the code here is the same as the one in line 595. Lets please use a function to do the common code

Reply:
I have replaced the literal strings with constants.
I have created a separate ticket for some further refactoring: https://jira.developmentgateway.org/browse/IATIIMPORT-201